packages/dm-core/src/components/Pickers/Datepicker/Calendar.tsx

```diff
@@ -178,8 +183,18 @@ export const Calendar = (props: CalendarProps): ReactElement => {
                       ? ''
                       : 'text-slate-400'
                 }
-                ${isHighlighted ? 'font-bold underline' : ''}
               `}
+                style={
```

Comment:
maybe use tailwind instead?

Reply:
doesn't work, need to override button classes, so thus we need style direct